### PR TITLE
Pressespiegel generation

### PR DESCRIPTION
**Pull request by bashvi**
Fixing #30 
I did NOT port over all the Presseartikel. I ported some for testing purposes. Someone else can do that... Sorry

### CODE REVIEW

File: content/de/pressreview/2014_10_18_20min.md [hunk 1]
```diff
@@ -0,0 +1,6 @@
+Title: Was macht die DDR-Flagge im Überwachungs-Büro?
+Slug: 2014_10_18_20min
+Date: 2014-10_18
+Lang: de
```
Comment (dbrgn) [inline content/de/pressreview/2014_10_18_20min.md]: This affects the language in which the content is displayed, right? So if we load the english version of the ccc-ch.ch website, it's not shown?

Can we get rid of this somehow in order to show the item in every language?

File: theme/templates/category.html [hunk 2]
```diff
@@ -4,31 +4,68 @@
...
   {% endif %}
 {% endblock %}
 
 {% block content_title %}
-{% if category == "medien" %}
+  {% if category == "medien" %}
     <h1>Medienmitteilungen</h1>
   {% else %}
-    <h1>Articles in the {{ category }} category</h1>
+    {% if category == "pressreview" %}
```
Comment (dbrgn) [inline theme/templates/category.html]: AFAIK you should be able to use elif, instead of a nested else / if block.

**Comment by dbrgn** [comment]
I think I'll already start coding the helper script mentioned in #30. Then we can use that to migrate the existing records.

File: content/de/pressreview/2014_10_18_20min.md (hunk 1 above)
Comment (bashvi) [inline content/de/pressreview/2014_10_18_20min.md]: Ah, thx. Changed it now

**Comment by dbrgn** [comment]
Looks good, as far as I can tell.

**Pull request merged**